pkgs/development/libraries/quarto/default.nix

```diff
@@ -11,14 +11,18 @@
 , makeWrapper
 , python3
 , extraPythonPackages ? ps: with ps; []
+
+# Allow overriding the Python and R used by Quarto
+, QUARTO_PYTHON ? "${python3.withPackages (ps: with ps; [ jupyter ipython ] ++ (extraPythonPackages ps))}/bin/python3"
+, QUARTO_R ? "${rWrapper.override { packages = [ rPackages.rmarkdown ] ++ extraRPackages; }}/bin/R"
```

Comment:
Similar comments apply as for Python.

Reply:
Likewise, I want to override this to `""`. The name of this argument is intentionally equivalent to the name of the environment variable that Quarto uses to determine where `R` is. Providing the string directly is semantically equivalent to setting the environment variable. I could, however, see an argument for adding an extra layer of override-ability where you can provide the `withPackages` arguments entirely and use that as the default here instead.